### PR TITLE
Add debug field names in sensor tab

### PR DESCRIPTION
**Pull request by haslinghuis**
1. Add debug field names to sensor tab
2. Refactor blackbox data fields to be maintained in one file only.
3. Add missing `RC_STATS` debug mode and fix unknown field names.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
Before refactoring:

image

**Review by chmelevskij — APPROVED** [review]

**Comment by HThuren** [comment]
nice work

**Pull request merged**
